src/docgen/view/documentation.ts

```diff
@@ -6,6 +6,7 @@ import * as glob from 'glob-promise';
 import * as reflect from 'jsii-reflect';
 import { TargetLanguage } from 'jsii-rosetta';
 import { transliterateAssembly } from 'jsii-rosetta/lib/commands/transliterate';
+import { bin as npmBin } from 'npm/package.json';
```

Comment:
So we are bundling npm into our lambda bundles. Is that the intent?

Reply:
That would be. However I don't actually think this can work with `esbuild` bundling, as the `bin/npmcli.js` file wouldn't be where it's supposed to be, I think... It'd need to be handled as external...

Reply:
So I guess that should be configured somehow, no?